lib/eu_central_bank.rb

```diff
@@ -136,6 +139,12 @@ def import_rates(format, s, opts = {})
     self
   end
 
+  def check_currency_available(currency)
+    return true if currency == "EUR"
+    return true if CURRENCIES.include?(currency)
```

Comment:
I'd suggest `currency.to_s` in both lines above, since somebody might try to pass a symbol